### DISCUSSION TITLE
Classroom Assignment bug: editing autograding autopoints doesn't keep values

### DISCUSSION THREAD

**Discussion by pbv** [community/community#67572] [GitHub Education]
### Hi everyone,

Hello everyone,
I just noticed that editing an individual assignment that had autograding tests that the points assigned to each passed test are not recorded - the field remains blank if I edit it again.
This was working (at least) last year when I used these same assignments.

**Comment by mh-skjelvareid** [comment]
I've had the same experience - I opened an issue about it today: https://github.com/education/autograding/issues/190
